waiter/src/waiter/scheduler/marathon.clj

```diff
@@ -276,19 +292,20 @@
 
 (defrecord MarathonScheduler [marathon-api mesos-api retrieve-framework-id-fn
                               home-path-prefix service-id->failed-instances-transient-store
-                              service-id->kill-info-store service-id->service-description
-                              force-kill-after-ms is-waiter-app?-fn]
+                              service-id->kill-info-store service-id->out-of-sync-state-store
```

Comment:
Why doesn't `service-id->out-of-sync-state-store` belong in ` sync-deployment-maintainer`?